api/resources_portal/migrations/0014_auto_20200812_2144.py

```diff
@@ -12,6 +12,12 @@ class Migration(migrations.Migration):
     ]
 
     operations = [
+        migrations.RenameField(
+            model_name="user", old_name="access_token", new_name="orcid_access_token",
+        ),
+        migrations.RenameField(
+            model_name="user", old_name="refresh_token", new_name="orcid_refresh_token",
+        ),
```

Comment:
I don't think you can modify old migrations like this. These have already been run on the staging server.